### PR TITLE
Switch to using tuple notation for routes

### PR DESCRIPTION
**Pull request by range-of-motion**

### CODE REVIEW

File: app/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public function map()
     protected function mapWebRoutes()
     {
         Route::middleware('web')
-             ->namespace($this->namespace)
+            //  ->namespace($this->namespace) // Disabled because we're using tuple notation for routes
```
Comment (Glennmen) [inline app/Providers/RouteServiceProvider.php:55]: No need to keep comments, you have git history to review what has been deleten and why should be in commit description?

File: routes/web.php [hunk 2]
```diff
@@ -65,38 +85,38 @@
...
+    Route::get('/spaces/{id}', SpaceController::class);
 
     Route::name('ideas.')->group(function () {
-        Route::get('/ideas/create', 'IdeaController@create')->name('create');
-        Route::post('/ideas', 'IdeaController@store');
+        Route::get('/ideas/create', [IdeaController::class, 'create'])->name('create');
+        Route::post('/ideas', [IdeaController::class, 'store']);
     });
 
     Route::get('/translations', function () {
```
Comment (Glennmen) [inline routes/web.php:122]: TIP Try to avoid closures, this breaks laravel route caching.

Comment (range-of-motion) [reply routes/web.php:122]: #191 🙂

File: app/Providers/RouteServiceProvider.php (hunk 1 above)
Comment (range-of-motion) [reply app/Providers/RouteServiceProvider.php:55]: Fair enough, will get rid of this altogether.

**Pull request merged**